packages/blockchain-api/src/blockscout.ts

```diff
@@ -296,11 +309,13 @@ export class BlockscoutAPI extends RESTDataSource {
       makerAmount: {
         value: new BigNumber(inTransfer.value).dividedBy(WEI_PER_GOLD).toString(),
         currencyCode: inTransfer.token,
+        knownExchangeRates: getKnownExchangeRates(inTransfer, outTransfer),
         timestamp,
       },
       takerAmount: {
         value: new BigNumber(outTransfer.value).dividedBy(WEI_PER_GOLD).toString(),
         currencyCode: outTransfer.token,
+        knownExchangeRates: getKnownExchangeRates(outTransfer, inTransfer),
```

Comment:
To clarify, one of these is undefined and the other is the rate. Is that correct?